scripts/skill_auditor/validation.py

```
@@ -0,0 +1,39 @@
+        "yaml_delimiters",
+        "has_name",
+        "has_description",
+    ]
+
+    missing_keys = [key for key in required_keys if key not in metrics]
+
+    if missing_keys:
+        raise ValueError(
+            f"Metrics extraction incomplete. Missing required keys: "
+            f"{', '.join(missing_keys)}. This is likely a bug in metrics_extractor."
+        )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)